### DISCUSSION TITLE
frustrated...

### DISCUSSION THREAD

**Discussion by shepaug** [ImageMagick/ImageMagick#5464] [Help]
OK..I have seen Image Magick around for years but never really used or got involved etc.  NOW I decided to try downloading/installing.

To launch all I get are error messages.

I am on Windows 8.1 and a 64 bit system I think they call it.

Downloaded the latest .exe of 64 bit and error/error messages. Dll files are not there (my computer) etc.

I tryed a 32 bit variety and error.

I have started trying to install missing .dll's but that even a problem. Install where ? Like .jpeg62.dll   ???

![image-magick-1](https://user-images.githubusercontent.com/111689520/185791617-5bc31336-c4a3-40ed-8df0-c5a8b3932198.jpg)
![image-magick-2](https://user-images.githubusercontent.com/111689520/185791624-ee73ebd8-95b9-4c3e-94ec-43913658443c.jpg)
![image-magick-3](https://user-images.githubusercontent.com/111689520/185791634-9578b7ba-e046-4b7d-90bd-bddc83031b90.jpg)
 messages.

**Comment by dlemstra** [comment]
The filezilla and the jpeg62.dll error message have nothing to do with ImageMagick. Which installation options did you pick when installing ImageMagick? And what error message did you get with the 64-bit version?
